include/json/value.h

```diff
@@ -171,6 +171,9 @@ class JSON_API Value {
     CZString(ArrayIndex index);
     CZString(const char* cstr, DuplicationPolicy allocate);
     CZString(const CZString& other);
+#ifdef JSON_HAS_RVALUE_REFERENCES
+	CZString(CZString&& other);
+#endif
     ~CZString();
     CZString& operator=(CZString other);
```

Comment:
I was going to but since CZString provides a copy and swap assignment operator I don't think it's worthwhile, if `other`  refers to an rvalue it will be moved in to the by-value parameter and then swap would be just as efficient.